index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav>
+            
+            <a href="#Home" class="active"> My Logo</a>
+            <a href=" " class="icon">
+                <i class="fa-solid fa-bars fa-2x"></i>
+            </a>
+        </nav> 
+    </header>
+    <section class ="main-section" >
+        <div class="intro-part">
+            <h1 class="intro"> I'm Dev Mike <br>
+                Glad to see<br>you!</h1>
+            <p class="about"> I'm a software developer I can help you build a project a product,feature or
+                website Look through some
+                of my work and <br> experience!If you like what you see and have a project you need coded don't hestiate
+                to contact me</p>
+            <div class="socials">
+                <h2>LET'S CONNECT</h2>
+                <div class="icons">
+                    <ul class ="social-media">
+                    <ul><a href=""> <i class="fa-brands fa-twitter fa-1x"></i></a></ul>
+                    <ul><a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a></ul>
+                    <ul><a href=""> <i class="fa-brands fa-github fa-1x"></i></a></ul>
+                    <ul><a href=""> <i class="fa-brands fa-m fa-1x"></i></a></ul>
+                    <ul><a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a></ul>
+                    </ul>
+                    
+                </div>
```

Comment:
[OPTIONAL] Your design is different from the figma since your background image is repeating. You can consider adding `background-size: cover;` and/or `background-repeat: no-repeat;`to fix this anomaly. :pray: